app/controllers/content_controller.rb

```diff
@@ -2,11 +2,7 @@ class ContentController < Api::BaseController
   before_action :validate_params!
 
   def show
-    @content = Queries::FindContent.retrieve(
-      from: from,
-      to: to,
-      organisation_id: params[:organisation_id]
-    )
+    @content = Queries::FindContent.retrieve(api_request: api_request)
```

Comment:
I feel that using an API request object obsfactates what `FindContent.retrieve` actually needs.

Reply:
Good point, actually it doesn't even need to do that.
Fixed now.

Reply:
It's all in a name.  `retrieve` now takes `content_filters`.